tests/unit/workflows/test_workflows_tasks_submission.py

```diff
@@ -47,417 +45,37 @@
-        result = obj.extra_data
-
-        assert expected == result
-
-        expected = u'Ticket 1 created:\nφοο'
-        result = obj.log._info.getvalue()
-
-        assert expected == result
-
-
-@patch('inspirehep.modules.workflows.tasks.submission.User')
-@patch('inspirehep.modules.workflows.tasks.submission.get_instance')
-def test_reply_ticket_thanks_the_user_when_their_literature_submission_is_received(mock_get_instance, mock_user):
-    mock_rt = MockRT()
-    mock_rt.create_ticket(Queue='Test', Status='new')
-    mock_get_instance.return_value = mock_rt
-    mock_user.query.get.return_value = MockUser('user@example.com')
-
-    schema = load_schema('hep')
-    subschema = schema['properties']['titles']
-
-    data = {
-        'titles': [
-            {'title': 'Partial Symmetries of Weak Interactions'},
-        ],
-    }
-    extra_data = {'ticket_id': 1}
-    assert validate(data['titles'], subschema) is None
-
-    obj = MockObj(data, extra_data)
-    eng = MockEng()
-
-    _reply_ticket = reply_ticket(
-        template='literaturesuggest/tickets/user_submitted.html',
-        context_factory=literaturesuggest_reply_ticket_context,
-        keep_new=True,
-    )
-
-    assert _reply_ticket(obj, eng) is None
-
-    expected = (
-        'Dear user@example.com,\n '
-        '\n '
-        'Thank you very much for suggesting "Partial Symmetries of Weak Interactions" '
-        'to INSPIRE. It has been received and will be reviewed by our staff as '
-        'soon as possible. You will hear back from us shortly.\n '
-        '\n '
-        'Thank you for sharing with INSPIRE!'
-    )
-    result = mock_rt.get_ticket(1)
-
-    assert expected == result['text']
+from mock import patch
 
-    expected = 'new'
-    result = mock_rt.get_ticket(1)
+from mocks import AttrDict, MockEng, MockFiles, MockObj, MockUser
 
-    assert expected == result['Status']
+import pytest
 
 
 @patch('inspirehep.modules.workflows.tasks.submission.User')
-@patch('inspirehep.modules.workflows.tasks.submission.get_instance')
-def test_reply_ticket_thanks_user_when_their_literature_submission_is_accepted(mock_get_instance, mock_user):
-    mock_rt = MockRT()
-    mock_rt.create_ticket(Queue='Test', Status='new')
-    mock_get_instance.return_value = mock_rt
+@patch('inspirehep.modules.workflows.tasks.submission.tickets.reply_ticket_with_template')
+@patch('inspirehep.modules.workflows.tasks.submission.rt_instance')
+def test_reply_ticket_calls_tickets_reply_with_template_when_template_is_set(mock_rt_instance, mock_reply_ticket_with_template, mock_user):
     mock_user.query.get.return_value = MockUser('user@example.com')
-
-    schema = load_schema('hep')
-    subschema = schema['properties']['titles']
-
+    mock_rt_instance = True
     data = {
         'titles': [
             {'title': 'Partial Symmetries of Weak Interactions'},
         ],
     }
     extra_data = {'ticket_id': 1}
-    assert validate(data['titles'], subschema) is None
```

Comment:
Actually you were right in removing this, because your implementation does no longer depend on the title being there. BTW, I'm going to merge this guy this weekend, so that you can continue working on this from Monday.